WcaOnRails/config/locales/en.yml

```diff
@@ -405,3 +405,16 @@ en:
       country: "Country"
       podiums: "Podiums"
       no_persons_found: "No person matches the given criteria."
+  committees:
+    new:
+      create_committee: "Create New Committee"
+    committee_form:
+      supports_md_html: "Supports <a href='https://daringfireball.net/projects/markdown/basics' target='_blank'>Markdown</a>"
```

Comment:
@viroulep do you know how to avoid duplications like this one? Is there a way to make an alias or something? Repeating will be painful for translators.